### PR TITLE
feat: Ingest plugin to store the application/API names

### PR DESCRIPTION
**Pull request by aelamrani**
fix gravitee-io/issues#2379

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+= Gravitee.io ingest plugin for ElasticSearch
+
+== Description
+
+This plugin allow you to enhance the ElasticSearch data schema as the API name / application name.
```
Comment (brasseld) [inline README.adoc]:
```suggestion
This plugin allow you to enhance the ElasticSearch data schema with the API name / application name.
```

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The plugin get these additional data from the Gravitee.io Management REST API.
+A cache mechanism allows to call the API only once as these data not change often. Default time to live is set to 24 hours.
+
+== Configuration
+
+=== From Gravitee.io
+
+As a new ES processor is available, we have to enable it from the gateway configuration ont the `gravitee.yml`:
```
Comment (brasseld) [inline README.adoc]:
```suggestion
As a new ES processor is available, we have to enable it from the gateway configuration on the `gravitee.yml`:
```

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+```yml
+gravitee.reporters.elasticsearch.pipeline.plugins.ingest=gravitee
+```
+
+=== From Elasticsearch
+
+First of all we have to install the plugin on ElasticSearch:
```
Comment (brasseld) [inline README.adoc]:
```suggestion
First of all you have to install the plugin within ElasticSearch:
```

**Comment by aelamrani** [comment]
> Please add a section about supported ES versions.

The is built for only one version..

**Comment by brasseld** [comment]
You mean a specific 6.8.0 version, or any 6.x ?

Also, should we have to include this page to docs.gravitee.io ?

**Comment by aelamrani** [comment]
> You mean a specific 6.8.0 version, or any 6.x ?
> 
> Also, should we have to include this page to docs.gravitee.io ?

6.8.0

**Pull request merged**